modules/log-collector-resources-scheduling.adoc

```diff
@@ -6,51 +6,84 @@
 [id="log-collector-resources-scheduling_{context}"]
 = Configuring resources and scheduling for logging collectors
 
-Administrators can modify the resources or scheduling of the collector by creating a `ClusterLogging` custom resource (CR) that is in the same namespace and has the same name as the `ClusterLogForwarder` CR that it supports.
-
-The applicable stanzas for the `ClusterLogging` CR when using multiple log forwarders in a deployment are `managementState` and `collection`. All other stanzas are ignored.
+Administrators can modify the resources or scheduling of the collector by configuring the `collector` field in a `ClusterLogForwarder` custom resource (CR).
```

Comment:
I think this is an `and` relationship (as in you can do both) and not an `or ` one (as in one or the other).